### PR TITLE
Support CentOS 8 aarch64 for Raspberry Pi

### PR DESCRIPTION
**Pull request by fisuda**
This PR updates documentation to support CentOS 8 aarch64 for Raspberry Pi. This is a way that Orion is run in a docker container on Raspberry Pi. It would be great if you could review this PR.

The test logs are the followings:
https://github.com/fisuda/report/tree/master/orion/20210325_centos8_aarch64

In the function test, one test has failed as shown:

- 1697_more_than_one_bad_input

This test may pass or fail. It seems that the issue is dependent on the timing.  This behavior is the same as before (https://github.com/telefonicaid/fiware-orion/pull/3630).

- more_than_one_bad_input.diff

```
VALIDATION ERROR: input file and reference file have different line count
```

- Expected (more_than_one_bad_input.regexpect)

```
3. grep SUMMARY in log-file (4 transactions, 1 bad input)
==========================================================
Transactions: REGEX((4 _new: 4_|5 _new: 5_))
DB status: ok, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Notification failure active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Bad input active alarms: 1, raised: _total: 1, new: 1_, released: _total: 0, new: 0_

```

- Actual (more_than_one_bad_input.out)

```
03. grep SUMMARY in log-file (4 transactions, 1 bad input)
==========================================================
Transactions: 0 _new: 0_
DB status: ok, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Notification failure active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Bad input active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Transactions: 4 _new: 4_
DB status: ok, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Notification failure active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
Bad input active alarms: 1, raised: _total: 1, new: 1_, released: _total: 0, new: 0_
```

Related PR: https://github.com/telefonicaid/fiware-orion/pull/3622, https://github.com/telefonicaid/fiware-orion/pull/3630

### CODE REVIEW

File: doc/manuals/admin/build_source.md [hunk 1]
```diff
@@ -85,12 +85,12 @@ The Orion Context Broker comes with a suite of unit, valgrind and end-to-end tes
         tar xfvj gmock-1.5.0.tar.bz2
         cd gmock-1.5.0
         ./configure
-        make
         sed -i 's/env python/env python2/' gtest/scripts/fuse_gtest_files.py  # little hack to make installation to work on CentOS 8
+        make
```
Comment (fisuda) [inline doc/manuals/admin/build_source.md:89]: In the case of the procedure before this update, the following error occurred.
```
(cd lib/.libs && rm -f libgtest_main.la && ln -s ../libgtest_main.la libgtest_main.la)
mkdir -p "./fused-src"
chmod -R u+w "./fused-src"
rm -f "./fused-src/gtest/gtest-all.cc"
rm -f "./fused-src/gtest/gtest.h"
"./scripts/fuse_gtest_files.py" "./fused-src"
env: 'python': No such file or directory
make[1]: *** [Makefile:1230: fused-gtest] Error 127
make[1]: Leaving directory '/gmock-1.5.0/gtest'
make: *** [Makefile:715: all-recursive] Error 1
```

File: docker/raspberry_pi.md [hunk 2]
```diff
@@ -41,34 +41,28 @@ sudo add-apt-repository \
...
-
 ## How to build Orion
 
 To build the docker images of Orion, Clone the Orion repository and run the `docker build` command.
 
 ```
 git clone https://github.com/telefonicaid/fiware-orion.git
 cd fiware-orion/docker
-docker build --build-arg=centos7 -t orion .
+docker build -t orion .
```
Comment (fisuda) [inline docker/raspberry_pi.md:65]: In the case of CentOS 8, there are docker images with the same tag `centos8.3.2011` for amd64 and aarch64. So the procedure  to build a docker image is the same.

https://hub.docker.com/layers/centos/library/centos/centos8.3.2011/images/sha256-7723d6b5d15b1c64d0a82ee6298c66cf8c27179e1c8a458e719041ffd08cd091?context=explore

File: doc/manuals/admin/build_source.md (hunk 1 above)
Comment (fgalan) [reply doc/manuals/admin/build_source.md:89]: You are right. The right order of commands is as you have modified in this PR. Thanks for the fix!

NTC

File: docker/raspberry_pi.md (hunk 2 above)
Comment (fgalan) [reply docker/raspberry_pi.md:65]: Ok. Nice.

NTC

**Review by fgalan — APPROVED** [review]
LGTM. Thanks!

I'll merge this PR and move the issue more_than_one_bad_input.test to a separate issue: https://github.com/telefonicaid/fiware-orion/issues/3828

**Pull request merged**